Tests/SentryTests/SentrySessionTests.m

```diff
@@ -26,7 +26,7 @@ - (void)testInitDefaultValues
 
 - (void)testSerializeDefaultValues
 {
-    SentrySession *expected = [[SentrySession alloc] initWithReleaseName:@"1.0.0"];
+    SentrySession *expected = [[SentrySession alloc] initWithReleaseName:@"1.0.0" distinctId:@""];
```

Comment:
`l`: I would use something different than an empty string to improve the quality of the test. The same applies for all the other tests.
```suggestion
    SentrySession *expected = [[SentrySession alloc] initWithReleaseName:@"1.0.0" distinctId:@"some-id"];
```